packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -196,6 +197,22 @@ export class MirrorNodeClient {
         throw new MirrorNodeClientError(error.message, effectiveStatusCode);
     }
 
+    async getPaginatedResults(url: string, pathLabel: string, resultProperty: string, allowedErrorStatuses?: number[], requestId?: string, results = [], page = 1) {
```

Comment:
There is already `pageAllResult` method for this functionality. It's done via iteration, while this one is done via recursion. Can you decide which one we can use for all operations needing pagination and remove the other one ?

Reply:
Would also prefer just one solution here. Use iteration for simplicity.

Reply:
Update `pageAllResult` to use this improved logic that takes in a mirror endpoint etc.
Let's open a ticket to handle this in 0.14.
For now let's get this into 0.13 as is since it's functional